From: Ansgar Burchardt <ansgar@43-1.org>
Date: Wed, 03 Mar 2010 23:04:10 +0900
Forwarded: https://rt.cpan.org/Public/Bug/Display.html?id=55184
Subject Fix POD syntax error

This patch fixes the following POD error:

  Around line 1389:
  You can't have =items (as at line 1399) unless the first thing after
  the =over is an =item

--- libpdf-create-perl.orig/lib/PDF/Create.pm
+++ libpdf-create-perl/lib/PDF/Create.pm
@@ -1386,8 +1386,6 @@
 
 =head2 URI links
 
-=over 5
-
 URI links have two components, the text or graphics object and the area
 where the mouseclick should occur.
 
@@ -1396,6 +1394,8 @@
 To define the click-sensitive area and the destination URI you use the
 C<annotation()> method.  
 
+=over 5
+
 =item * annotation([parameters])
 
 Define an annotation. This is a sensitive area in the PDF document where
